vector/v.outlier/v.outlier.html

```diff
@@ -13,12 +13,12 @@ <h2>DESCRIPTION</h2>
 (default), or only positive or only negative outliers. Filtering out
 only positive outliers can be useful to filter out vegetation returns
 (e.g. from forest canopies) from LIDAR point clouds, in order to
-extract Digital Terrain Models. Filtering out only negative outliers
+extract digital terrain models. Filtering out only negative outliers
 can be useful to estimate vegetation height.
 
 <p>
-There is a flag to create a vector that can be visualizated by
-qgis. That means that topology is build and the z coordinate is
+There is a flag to create a vector that can be visualized in
+QGIS. That means that topology is build and the z coordinate is
```

Comment:
```suggestion
QGIS. That means that topology is built and the z coordinate is
```
Did I understand the sentence right?

Reply:
I missed that one, "built", should be. Thanks.